examples/access-token-server/README.md

```diff
@@ -41,6 +51,18 @@ SERVICE_ACCOUNT_CREDENTIALS=
    - Request header - `Authorization: Bearer access_token`
 4. Ensure this servers endpoint is defined as an entitlement for the extension.
 
+## Docker container instructions
+
+1. Build the image locally: ./docker-build.sh
```

Comment:
Is this from the repo root or the example/access-token-server directory? I guess it won't be found if you're not in the right place.

Reply:
yeah, that's meant to be relative to the README